### DISCUSSION TITLE
Request for Support: For Better Localization Experience, We Request the Release of a Chinese Version!

### DISCUSSION THREAD

**Discussion by lein2023** [Anjok07/ultimatevocalremovergui#550] [Ideas]
Translation: Localization is extremely important! Hey there! Our project is very popular in China and has attracted many users. We hope that the technical team can provide a Chinese interface for our Chinese users, but we are unsure of how to display Chinese characters. Would it be possible to have a Chinese version? If needed, I am happy to assist with translation, either personally or by using Google Translate. Is anyone interested in providing help?

**Comment by hua81939554** [comment]
强烈支持，我就是中国来的

**Comment by ybhka2022** [comment]
Strong support, need Chinese version interface

**Comment by taozhiyu** [comment]
I add i18n in most places, but NOT widely verified yet.

and I DO NOT know how to build a release exe file.

If anyone could give me some suggestions I'll be very grateful.

Here's my forked repo below:

https://github.com/taozhiyu/ultimatevocalremovergui_zh

**Comment by lein2023** [comment]
I have received your mailing. If I were online, I would watch it. Please wait for a while. [Robot response]
